File: MultiScreen/app/src/main/java/com/example/multiscreen/MainActivity.java
package com.example.multiscreen;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;


public class MainActivity extends AppCompatActivity {
    public static final String BNJ = "com.example.multiscreen.ORDER";


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void placeyourorder (View view){

        Intent intent = new Intent(this, OrderActivity.class);
        EditText editText1 = findViewById(R.id.editTextTextPersonName);
        EditText editText2 = findViewById(R.id.editTextTextPersonName2);
        EditText editText3 = findViewById(R.id.editTextTextPersonName3);
        String message = "Order for " + editText1.getText().toString() + ", " + editText2.getText().toString() + " & " + editText3.getText().toString() + " Has been successfully placed!";
        intent.putExtra(BNJ, message);
        startActivity(intent);






    }
}

File: ToDoList/app/src/main/java/com/example/to_dolist/MainActivity.kt
package com.samderlust.todoappjava

import android.R
import android.os.Bundle
import android.view.View
import android.widget.ArrayAdapter
import android.widget.Button
import android.widget.EditText
import android.widget.ListView
import androidx.appcompat.app.AppCompatActivity
import java.util.*

class MainActivity : AppCompatActivity() {
    var button: Button? = null
    var inputText: EditText? = null
    var listView: ListView? = null
    var list: ArrayList<String>? = null
    fun onClickAdd(v: View?) {
        val text = inputText!!.text.toString()
        list!!.add(text)
        val adapter: ArrayAdapter<*> = ArrayAdapter(this, R.layout.simple_list_item_1, list)
        listView!!.adapter = adapter
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button = findViewById(R.id.addButton)
        inputText = findViewById(R.id.inputText)
        listView = findViewById(R.id.listView)
        list = ArrayList()
    }
}